### PR TITLE
clarifying config file options

### PR DESCRIPTION
**Pull request by beckettsean**

### CODE REVIEW

**Comment by beckettsean** [comment]
To address https://github.com/influxdb/influxdb/issues/3543#issuecomment-127460160

File: content/docs/v0.9/introduction/installation.md [hunk 1]
```diff
@@ -58,12 +58,25 @@ For users who don't want to install any software and are ready to use InfluxDB,
...
+```shell
+/opt/influxdb/influxd -config /etc/influxdb/influxdb.generated.conf
 ```
 
+If no `-config` option is supplied, InfluxDB will use the default configuration file that ships with the installer. The actual location of the file depends on your OS. It is an identical file to the generated configuration file except that the default file has multiple comments.
+
+> Note: The `influxd` command has two similarly named flags. The `config` flag prints a generated default configuration file to STDOUT but does not launch the `influxd` process. The `-config` flag takes a single argument, which is the path to the InfluxDB configuration file to use when launching the process.
+
+
 ## Development Versions
```
Comment (mjdesa) [inline content/docs/v0.9/introduction/installation.md]: If you start influxdb with `sudo service influxdb start` where does your config file need to be?

Comment (beckettsean) [inline content/docs/v0.9/introduction/installation.md]: I don't know. I was hoping that @otoolep or someone would know.

File: content/docs/v0.9/introduction/installation.md [hunk 2]
```diff
@@ -58,12 +58,25 @@ For users who don't want to install any software and are ready to use InfluxDB,
...
 ```
-/opt/influxdb/influxd config
+
+Edit the `influxdb.generated.conf` file to have the desired configuration settings. When launching InfluxDB, point the process to the correct configuration file using the `-config` option.
+
+```shell
+/opt/influxdb/influxd -config /etc/influxdb/influxdb.generated.conf
 ```
 
+If no `-config` option is supplied, InfluxDB will use the default configuration file that ships with the installer. The actual location of the file depends on your OS. It is an identical file to the generated configuration file except that the default file has multiple comments.
```
Comment (gunnaraasen) [inline content/docs/v0.9/introduction/installation.md]: I think there's also another option, which is running `influxd` with no config provided at all. In that case, InfluxDB will use the built-in defaults, which should be the same as if the default values generated by `influxd -config`.

Comment (gunnaraasen) [inline content/docs/v0.9/introduction/installation.md]: Actually, I tested this and my comment is not true. `influxd` gives an error when no `-config` option is given.

Comment (gunnaraasen) [inline content/docs/v0.9/introduction/installation.md]: We should probably note that running `influxd` with no command is the same as running the `influxd run` command (which will return an error if the `-config .

**Pull request merged**